core/src/com/unciv/ui/screens/pickerscreens/ImprovementPickerScreen.kt

```diff
@@ -181,7 +196,7 @@ class ImprovementPickerScreen(
             improvementButton.onActivation(type = ActivationTypes.Tap, noEquivalence = true) {
                 selectedImprovement = improvement
                 pick(improvement.name.tr())
-                descriptionLabel.setText(improvement.getDescription(ruleSet))
+                descriptionLabel.setText(improvement.getDescription(ruleset))
             }
 
             improvementButton.onDoubleClick { accept(improvement) }
```

Comment:
(4 lines below)
Can we enable the button when it is queable as well? Maybe changing the color would also be helpful.